pkg/activator/net/revision_backends.go

```diff
@@ -161,7 +231,9 @@ func (rw *revisionWatcher) checkDests() {
 		healthStates[dh.dest] = dh.health
 	}
 
+	rw.logger.Infof("Done probing, got healthStates %v", healthStates)
 	if !reflect.DeepEqual(rw.healthStates, healthStates) {
+		rw.logger.Infof("sending update")
```

Comment:
There is no format string, so `Info` instead of `Infof`?   Probably need a better log message here too.

Reply:
This was leftover debugging, removed it altogether